src/main/java/nl/tudelft/contextproject/model/Drawable.java

```diff
@@ -8,7 +8,7 @@
  * Interface representing a Drawable object.
  * This object has a geometry that can be updated.
  */
-public interface Drawable {	
+public interface Drawable extends Collidable {	
```

Comment:
Hmm, I disagree with chiel then. It stands to reason that every object we draw to the screen COULD be collidable, but does not mean that every object IS collidable.
This method removes the possibility of using `if (something instanceof Collidable)` to check if something collides with it.